install_config/configuring_sdn.adoc

```diff
@@ -289,3 +315,45 @@ flannel_interface=eth0
 . Optionally, you can specify the interface to use for inter-host communication
 using the `flannel_interface` variable. Without this variable, the
 {product-title} installation uses the default interface.
+
+NOTE: In order to provide custom networking CIDR for pods and services using
+flannel, there is a bugzilla being invested by Red Hat Bugzilla ID 1473858 at
+the time of writing this paper.
+
+To isolate container traffic from the rest of the {product-title} traffic, it is
+recommended to create an isolated tenant network and attach all the nodes to it.
+If using a different network interface (eth1), remember to configure the
+interface to start at boot time as:
+*_/etc/sysconfig/network-scripts/ifcfg-eth1_* file:
+----
+DEVICE=eth1
+TYPE=Ethernet
+BOOTPROTO=dhcp
+ONBOOT=yes
+DEFTROUTE=no
+PEERDNS=no
+----
+
+=== Flannel iptables rules
+In order to enable flannel, a set of iptables rules should be added as well
+after the {product-title} installation has been completed.
+
+On every {product-title} node:
+
+----
+iptables -A DOCKER -p all -j ACCEPT
+iptables -t nat -A POSTROUTING -o eth1 -j MASQUERADE
```

Comment:
Why using eth1 here? it should be eth0 if we use the default linux network?

Reply:
To be honest, I don't know about using just 1 interface for everything. My experience is with the OCP on OSP ref. arch. where this is a requisite. Maybe @markllama or @rlopez133 can clarify.

Reply:
In the refarch it is a separate interface to keep the container traffic separate from the control and inbound.  This is the refarch design developed with the OCP networking team quite a while ago.